### ISSUE TITLE
Somehow use the pseudo-button with current frame size

### ISSUE THREAD

**Issue by pavel-the-best** [flameshot-org/flameshot#997]
My proposal it should select between several ratios (16:9, 1:1, 2:1, etc.) or fix current ratio.

**Comment by mmahmoudian** [comment]
@pavel-the-best Alt. Title: "lock aspect ratio button" ?

**Comment by pavel-the-best** [comment]
No, I'm talking about removing confusion when clicking on that "button" and getting nothing.
My solution is just one of possible ones

**Comment by mmahmoudian** [comment]
@pavel-the-best I remember that @borgmanJeremy wanted to improve this. We had [a brief discussion on Slack about this on 2020.09.18](https://flameshotworkspace.slack.com/archives/C019W6EPYMN/p1600443991007300).

**Comment by mxmilkiib** [comment]
Having an option in Area menu in the Launcher to choose 16:9, 9:16, etc., and then have the selection area only resize to that ratio, would be amazing.
